suite-common/wallet-config/src/networksConfig.ts

```diff
@@ -239,6 +239,40 @@ export const networks = {
         tradeCryptoId: 'optimistic-ethereum--0x0000000000000000000000000000000000000000',
         caipId: 'eip155:10',
     },
+    avax: {
+        symbol: 'avax',
+        displaySymbol: 'AVAX',
+        displaySymbolName: 'Avalanche',
+        name: 'Avalanche C-Chain',
+        networkType: 'ethereum',
+        chainId: 43114,
+        bip43Path: "m/44'/60'/0'/0/i",
+        decimals: 18,
+        testnet: false,
+        explorer: getExplorerUrls('https://snowscan.xyz/', 'ethereum'),
+        features: [
+            'rbf',
+            'sign-verify',
+            'tokens',
+            'nfts',
+            'coin-definitions',
+            'nft-definitions',
```

Comment:
If `nft-definitions` are supported, `release-suite-definitions.yml` should also have the command to update those right?

Reply:
uff, true, thank you